realm/src/androidTest/java/io/realm/RealmConfigurationTest.java

```diff
@@ -343,4 +342,18 @@ public void testDifferentSchemasThrows() {
             realm1.close();
         }
     }
+
+    public void testDifferentDurabilityThrows() {
+        RealmConfiguration config1 = new RealmConfiguration.Builder(getContext()).inMemory().build();
+        RealmConfiguration config2 = new RealmConfiguration.Builder(getContext()).inMemory().build();
```

Comment:
This is the same configuration :). One of them should leave out the `inMemory()` setting.